src/common/transformations/src/transformations/common_optimizations/random_uniform_fusion.cpp

```diff
@@ -26,10 +27,8 @@ ov::pass::RandomUniformFusion::RandomUniformFusion() {
         {data_pattern, ru_min_input_pattern, ru_max_input_pattern},
         pattern::consumers_count(1));
     const auto const_pattern = ov::pass::pattern::wrap_type<ov::op::v0::Constant>();
-
-    const auto convert_pattern = ov::pass::pattern::wrap_type<ov::op::v0::Convert>({random_uniform_pattern});
     const auto random_uniform_or_convert_pattern =
```

Comment:
I would recommend update name from "random_uniform_or_convert_pattern" to "optional_convert"

Reply:
applied